### ISSUE TITLE
Request for running PyMFEM build test in CI

### ISSUE THREAD

**Issue by sshiraiwa** [mfem/mfem#3652]
Hi Team,
I propose to run a build-test of Python wrapper when something is pushed to master.
If you have MPI and Python in a test environment, this could be done by something like.
```
pip install numpy mpi4py six swig
git clone git@github.com:mfem/PyMFEM.git
cd PyMFEM
python setup.py install --with-parallel --mfem-branch=master
cd test
python run_examples.py -serial -verbose  -ex ex1
python run_examples.py -parallel -verbose -ex ex1p
```
There is also this
https://github.com/mfem/PyMFEM/blob/master/.github/workflows/test_with_MFEM_master.yml, 
which build PyMFEM build in a various setting. We could modify it to call this when a change is pushed to master.

**Comment by tzanio** [comment]
I think that's a good idea @sshiraiwa, do you want to propose a PR?

**Comment by v-dobrev** [comment]
It's probably best to run this not just on `master` but all PRs -- that way we can catch issues before they are merged in `master`. To do that, we'll need to replace `--mfem-branch=master` with a local MFEM directory. Is that supported in the PyMFEM installer?

**Comment by sshiraiwa** [comment]
@v-dobrev , @tzanio let me look at this this weekend. Meantime, please point me a CI script which you think is good to add a call to test the wrapper.

**Comment by v-dobrev** [comment]
Hi @sshiraiwa,

It's probably easiest to use a file like https://github.com/mfem/mfem/blob/master/.github/workflows/builds-and-tests.yml as a starting point -- copy it with a different name and edit that. A lot of the things in that file can probably be removed.

**Comment by v-dobrev** [comment]
@sshiraiwa, @justinlaughlin, this was already addressed in #4283, right?

If not -- feel free to reopen.
